AlphaWallet/Tokens/Types/UniswapHolder.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  UniswapHolder.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 21.08.2020.
+//
+
+import UIKit
+
+struct UniswapHolder {
+    private static let baseURL = "https://app.uniswap.org/#"
+
+    let input: Input
+    var version: Version = .v2
+    var theme: Theme = .dark
+    var method: Method = .swap
+
+    var url: URL? {
+        var components = URLComponents()
+        components.path = method.rawValue
+        components.queryItems = [
+            URLQueryItem(name: Version.key, value: version.rawValue),
+            URLQueryItem(name: Theme.key, value: theme.rawValue)
+        ] + input.urlQueryItems
+
+        //NOTE: URLComponents doesn't allow path to contain # symbol
```

Comment:
> What implication does this have here?

i tried to build URL through URLComponents, but all the time it failed because path contained symbol "#", and converted it to special URL symbol.  that is why link was not work.